tests/codebuild/run_canarytest.sh

```diff
@@ -8,17 +8,24 @@ function cleanup {
     # We want to run every command in this function, even if some fail.
     set +e
 
-    echo "Controller manager logs:"
-    kubectl -n sagemaker-k8s-operator-system logs "$(kubectl get pods -n sagemaker-k8s-operator-system | grep sagemaker-k8s-operator-controller-manager | awk '{print $1}')" manager
-
     # Describe, if the test fails the Additional field might have more helpful info.
-    echo "trainingjob description:"
-    kubectl describe trainingjob
+    # This is helpful for nonEphemeral cluster since we are not printing manager pod logs
+    echo "Job descriptions:"
+    kubectl describe TrainingJob
+    kubectl describe ProcessingJob
+    kubectl describe HyperparameterTuningJob
+    kubectl describe BatchTransformJob
+    kubectl describe HostingDeployment
+    kubectl describe HostingAutoscalingPolicy
 
-    delete_all_resources
+    delete_all_resources "${crd_namespace}"
 
     if [ -z "${USE_EXISTING_CLUSTER}" ]
     then
+        # Managable to print logs for ephemeral cluster
+        get_manager_logs "${crd_namespace}"
```

Comment:
I think this will fail. `get_manager_logs` expects you to pass the namespace of the controller, not the namespace of the CRDs. You can probably just leave that empty and let it assume the default namespace since I don't think we change the controller namespace.

Reply:
Actually I see below that this is the namespace of the controller. Perhaps we can have a different variable for the controller vs. the CRDs?

Reply:
right the variable naming was bad. Changed it